CMakeLists.txt

```diff
@@ -192,7 +192,10 @@ add_definitions("-DHIP_COMPILER_FLAGS=${HIP_COMPILER_FLAGS}")
 
 
 # HIP
-if( MIOPEN_BACKEND STREQUAL "HIP" OR MIOPEN_BACKEND STREQUAL "HIPOC")
+if( MIOPEN_BACKEND STREQUAL "HIP" OR MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN_BACKEND STREQUAL "NOGPU")
+    if( MIOPEN_BACKEND STREQUAL "NOGPU")
+        set(MIOPEN_BACKEND_NOGPU 1)
```

Comment:
This naming gives the impression that `MIOPEN_BACKEND_NOGPU` and `MIOPEN_BACKEND_HIP` have equal "rights", but it is not so.

Please rename this to `MIOPEN_BACKEND_MODE_NOGPU` or delete (it is unused anyway).

Reply:
I have renamed the define, however, it is not useless since I use it in the client application to detect if the No GPU mode was enabled or not, which provides for runtime checks. Therefore, I need to keep it.

Reply:
Thanks for the explanation.